### ISSUE TITLE
Unable to access focusable elements inside a Popover2 with HOVER interactionKind 

### ISSUE THREAD

**Issue by evansjohnson** [palantir/blueprint#5851]
#### Code Sandbox
["Hover" button in "Interactions" example](https://blueprintjs.com/docs/#popover2-package/popover2.interactions)

#### Steps to reproduce
Tab to open hover button

#### Actual behavior
Unable to access menu items with keyboard navigation

#### Expected behavior
Should be able to tab to menu items without popover closing, or otherwise able to access menu items with keyboard navigation

#### Possible solution
Is the user expected to implement custom open control if accessibility is required? It would be nice to get this out of the box!

Note that `CLICK` interactionKind works as expected, perhaps if TAB key opens the popover it acts more like how the click action currently works.

**Comment by mm-wang** [comment]
Still in https://blueprintjs.com/docs/#core/components/popover.interactions
